radial_bar_chart_plugin/README.md

```diff
@@ -0,0 +1,190 @@
+# GoodData.UI Dashboard Plugin - Radial Bar Chart
+
+This plugin replaces all column charts with a custom [Recharts](https://recharts.org/) [radial bar chart](https://recharts.org/en-US/examples/SimpleRadialBarChart).
+
+This plugin was developed against https://jirizajic.demo.cloud.gooddata.com/, workspace id `ecommerce-parent`, dashoard id `c31f7bec-e820-4731-a451-974cd5d832c2`. This plugin should run against any dashboard on any workspace on any domain.
```

Comment:
Please describe how to setup this plugin (few steps, what to change, how and where)